src/ArrayObject.php

```diff
@@ -450,7 +450,10 @@ public function __isset($name)
      * {@inheritdoc}
      *
      * @throws \Rexlabs\ArrayObject\Exceptions\InvalidOffsetException
+     *
+     * @return mixed
      */
+    #[\ReturnTypeWillChange]
```

Comment:
@BlankaK found a way to avoid the php 7/8 issue that's caused by type hints in this case. There's a specific attribute that php 8 introduces to help deal with cross version compatibility issues.  https://www.php.net/manual/en/class.returntypewillchange.php